### PR TITLE
Added Metal renderer for macos-native.

### PR DESCRIPTION
**Pull request by Rsedaikin**

### CODE REVIEW

**Comment by olonho** [comment]
@Rsedaikin please refactor per current state.

**Comment by olonho** [comment]
Could you please actualize this PR?

File: skiko/src/macosMain/kotlin/org/jetbrains/skiko/SystemTheme.macos.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.skiko
+
+import platform.AppKit.*
```
Comment (olonho) [inline skiko/src/macosMain/kotlin/org/jetbrains/skiko/SystemTheme.macos.kt:3]: Why so wide exports?

**Review by olonho — APPROVED** [review]

File: skiko/src/macosMain/kotlin/org/jetbrains/skiko/SystemTheme.macos.kt (hunk 1 above)
Comment (Rsedaikin) [reply skiko/src/macosMain/kotlin/org/jetbrains/skiko/SystemTheme.macos.kt]: Done.

**Pull request merged**